service/service.go

```diff
@@ -234,27 +274,36 @@ func (s *Service) Start(b Broadcaster, atxProvider challenge_verifier.Verifier)
 	ctx, stop := context.WithCancel(context.Background())
 	s.stop = stop
 
-	s.SetBroadcaster(b)
-	s.SetChallengeVerifier(atxProvider)
+	s.broadcaster = b
 
+	var toResume []*round
```

Comment:
can you have multiple rounds ongoing?

Reply:
please file an issue for this to at least explore the correctness. 
recovering multiple rounds doesn't seem to fit within the poet design. is the code even capable of running multiple rounds at the same time?

Reply:
The code is capable of running multiple rounds at once. In this PR I want to preserve the existing behaviour. 

Created a ticket to track it: https://github.com/spacemeshos/poet/issues/182